content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -15,14 +15,14 @@ interface.
 
 {{< note >}}
 Starting from Kubernetes version 1.23, the kubelet supports the use of either `/` or `.`
-as separators for sysctl names. 
+as separators for sysctl names.
+Starting from Kubernetes version 1.25, setting Sysctls for a Pod and PodSecurityPolicy 
```

Comment:
I'm afraid that there is no PodSecurityPolicy API in Kubernetes v1.25 - see https://blog.k8s.io/2022/08/04/upcoming-changes-in-kubernetes-1-25/#a-note-about-podsecuritypolicy

Reply:
Thanks for reminding me. I will check how to change it.